spec/support/client_registry.rb

```diff
@@ -95,13 +95,19 @@ def new_global_client(name)
       )
     # Provides an authorized mongo client that retries writes.
     when 'authorized_with_retry_writes'
-      global_client('authorized').with(retry_writes: true).tap do |client|
+      global_client('authorized').with(
+        retry_writes: true,
+        server_selection_timeout: 4.97,
```

Comment:
How were these server selection timeouts (4.97 and 4.99) chosen? Given how specific they are, I think it makes sense to comment why they are what they are.

Reply:
They are numbers which can be traced to where in the code they are defined, as opposed to a number like `5` which is hard to grep for. The fractions chosen are arbitrary.